dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/ValidationExecution.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2023 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.dockstore.webservice.core.metrics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dockstore.webservice.core.metrics.constraints.ISO8601ExecutionDate;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.time.Instant;
+import java.util.Date;
+import java.util.Optional;
+import javax.validation.constraints.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is an object to encapsulate an execution that validates a workflow in an entity. Does not need to be stored in the database.
+ */
+@Schema(name = "ValidationExecution", description = "Metrics of a workflow validated on a platform", allOf = Execution.class)
+public class ValidationExecution extends Execution {
```

Comment:
Would it make sense to include information about *why* a validation failed - maybe a message, or a reason code, or something like that?  Not sure one way or the other, just throwing this out there...

Reply:
Hmm, I think Charles mentioned somewhere in this PR that as an author, the error message would be helpful. I'll add it